src/providers/osu.ts

```diff
@@ -0,0 +1,34 @@
+import { OAuthConfig, OAuthUserConfig } from "."
+
+export interface OsuProfile {
+  id: string
+  username: string
+  avatar_url: string
```

Comment:
Do you know what other fields exist here? Would be nice to add them as well, in case someone wonders and needs them!

Reply:
I added the UserCompact and User. Unfortunately the documentation is incorrect and partly not complete. So I added what was officially written down and omitted the unofficial fields.

Reply:
I understand, thank you!